spring-ai-docs/src/main/antora/modules/ROOT/pages/api/clients/mistralai-chat.adoc

```diff
@@ -68,13 +68,18 @@ The prefix `spring.ai.mistralai.chat` is the property prefix that lets you confi
 | spring.ai.mistralai.chat.enabled | Enable MistralAI chat client.  | true
 | spring.ai.mistralai.chat.base-url   | Optional overrides the spring.ai.mistralai.base-url to provide chat specific url |  -
 | spring.ai.mistralai.chat.api-key   | Optional overrides the spring.ai.mistralai.api-key to provide chat specific api-key |  -
-| spring.ai.mistralai.chat.options.model | This is the MistralAI Chat model to use | `mistral-tiny` (the `gpt-3.5-turbo`, `gpt-4`, and `gpt-4-32k` point to the latest model versions)
+| spring.ai.mistralai.chat.options.model | This is the MistralAI Chat model to use | `mistral-tiny` (the `mistral-small`, `mistral-medium`, and `mistral-large` point to the latest model versions)
```

Comment:
I've added those model names as enums (and tested them): https://github.com/spring-projects/spring-ai/blob/50547188f771dc957f26823c1b167b53f782af14/models/spring-ai-mistral-ai/src/main/java/org/springframework/ai/mistralai/api/MistralAiApi.java#L699

They come from the: https://docs.mistral.ai/platform/endpoints/#mistral-ai-generative-models
So perhaps we should stick with the officially listed ones?

Reply:
Yes, they come from the official list, I'll redefine the enums instead to get closer to the value that is a valid name.